cpp/src/parquet/statistics_test.cc

```diff
@@ -703,12 +709,16 @@ class TestStatisticsHasFlag : public TestStatistics<TestType> {
       EXPECT_TRUE(statistics2->HasMinMax());
       EXPECT_TRUE(encoded_stats2.has_min);
       EXPECT_TRUE(encoded_stats2.has_max);
+      EXPECT_TRUE(encoded_stats2.is_min_value_exact);
+      EXPECT_TRUE(encoded_stats2.is_max_value_exact);
```

Comment:
So this is a bit confusing and error-prone, see https://en.cppreference.com/w/cpp/utility/optional/operator_bool.html